dependencies.gradle

```diff
@@ -89,19 +88,19 @@ ext.dep = [
   "prometheusHttpserver": "io.prometheus:simpleclient_httpserver:0.8.1",
   "protobufGradlePlugin": "com.google.protobuf:protobuf-gradle-plugin:0.8.10",
   "protoc": "com.google.protobuf:protoc:3.6.1",
-  "retrofit": "com.squareup.retrofit2:retrofit:2.8.1",
-  "retrofitMoshi": "com.squareup.retrofit2:converter-moshi:2.8.1",
-  "retrofitProtobuf": "com.squareup.retrofit2:converter-protobuf:2.8.1",
-  "retrofitWire": "com.squareup.retrofit2:converter-wire:2.8.1",
+  "retrofit": "com.squareup.retrofit2:retrofit:2.9.0",
+  "retrofitMoshi": "com.squareup.retrofit2:converter-moshi:2.9.0",
+  "retrofitProtobuf": "com.squareup.retrofit2:converter-protobuf:2.9.0",
+  "retrofitWire": "com.squareup.retrofit2:converter-wire:2.9.0",
   "servletApi": "javax.servlet:javax.servlet-api:3.1.0",
-  "shadowJarPlugin": "com.github.jengelman.gradle.plugins:shadow:5.1.0",
+  "shadowJarPlugin": "com.github.jengelman.gradle.plugins:shadow:6.1.0",
   "slf4jApi": "org.slf4j:slf4j-api:1.7.28",
   "tink": "com.google.crypto.tink:tink:1.2.0",
-  "tracingDatadog": "com.datadoghq:dd-trace-api:0.67.0",
+  "tracingDatadog": "com.datadoghq:dd-trace-api:0.65.0",
```

Comment:
Downgrade?

Reply:
Seems like these need to be updated in `master-dependencies.json` for the dep-upgrade to not downgrade.

Reply:
See above